modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -22,11 +22,13 @@ def show
           end
         end
 
-        def appoint_organization
+        def submit2122
           validate_request!(ClaimsApi::V2::ParamsValidation::PowerOfAttorney)
           poa_code = parse_and_validate_poa_code
           unless poa_code_in_organization?(poa_code)
-            raise ::Common::Exceptions::UnprocessableEntity.new(detail: 'POA Code must belong to an organization.')
+            raise ::ClaimsApi::Common::Exceptions::Lighthouse::UnprocessableEntity.new(
+              detail: 'POA Code must belong to an organization.'
+            )
           end
 
           submit_power_of_attorney(poa_code)
```

Comment:
`submit_power_of_attorney(poa_code)` might need reworked/updated.

- `VBMSUpdater` doesn't exist, it got renamed
- I don't know if the `header_md5` stuff works with the new ICN url path instead of the V1 auth headers

Reply:
just getting to perusing this, but yes, the `header_md5` is working. It pulls from the EVSS Auth Headers we use to build the old request to EVSS. That being said, a majority of the data in that comes from the old headers in v1, but with our new v2 flow we pull some data from our request to MPI. 

This is being used in the lookup in the submit method
```
power_of_attorney = ClaimsApi::PowerOfAttorney.find_using_identifier_and_source(header_md5:, source_name:)
```

with that in mind, though... that looks for an existing Power of Attorney, but with the recent updates to LH API Standards, we should be creating a new entry for each request, even if it were to be identical. 
```
POST should not be idempotent; meaning, if the request is sent again, a second resource should be created.
```
so I'm not sure how relevant this lookup is anywho. Could possibly be useful as a troubleshooting method to find values but it seems like we should be creating on each request.